source/_patterns/02-organisms/07-quicksearch/00-quicksearch-image.mustache

```diff
@@ -0,0 +1,23 @@
+<div class="page-headline">
+  <div class="container">
+    <h1 class="qs-h1">{{ pageHeadline }}</h1>
+  </div>
+</div>
+
+<div class="qs-background">
+  <picture>
```

Comment:
picture-tag is not supported in ie11
https://caniuse.com/#search=picture

Do you have a fallback?

Reply:
there is a img tag as well. In any case, same thing we use for teasers